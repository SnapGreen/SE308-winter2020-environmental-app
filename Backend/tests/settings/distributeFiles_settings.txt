settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/scripts/settings.txt
	ZIPFILE: fakefile.zip
	DATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/
	RAWDATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/
	FDADIR: FDA/
	FDADATASOURCE: branded_food.csv
	FDAUPDATESOURCE: food_update_log_entry.csv
	LASTLATEST: FoodData_Central_branded_food_csv_2020-04-29.zip,202005010424,74439754
	RAWDATADEST: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/FDA/
	DATADEST: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/
	RAWDATAUPDATEDEST: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/FDA/202005010424/
	DATAUPDATEDEST: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/202005010424/
	USAGE:
		Usage: ./distributeFiles.sh <zipfile> [OPTION] (use option -h for help)
	HELP:
	This script distributes the contents of the provided FDA zipfile to
	the proper directories.  If there are no uploads left, it
	puts them in the upload directory.  Otherwise, it creates
	a new directory in both data/ and dataraw/ for the update
		Usage: ./distributeFiles.sh <zipfile> [OPTION] (use option -h for help)
			-b: bypass debug mode (don't keep temp files)
			-ns: output settings relative to repo root only
			-s: output settings only
			-t: test mode (silent)
			-h: print help
