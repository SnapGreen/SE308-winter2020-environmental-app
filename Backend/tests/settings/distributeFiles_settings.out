settings check:
	SETTINGS: settings.txt
	ZIPFILE: fakefile.zip
	DATADIR: ../data/
	RAWDATADIR: ../dataraw/
	FDADIR: FDA/
	FDADATASOURCE: branded_food.csv
	FDAUPDATESOURCE: food_update_log_entry.csv
	LASTLATEST: FoodData_Central_branded_food_csv_2020-04-29.zip,202005010424,74439754
	RAWDATADEST: ../dataraw/FDA/
	DATADEST: ../data/FDA/
	RAWDATAUPDATEDEST: ../dataraw/FDA/202005010424/
	DATAUPDATEDEST: ../data/FDA/202005010424/
	USAGE:
		Usage: ./distributeFiles.sh <zipfile> [OPTION] (use option -h for help)
	HELP:
	This script distributes the contents of the provided FDA zipfile to
	the proper directories.  If there are no uploads left, it
	puts them in the upload directory.  Otherwise, it creates
	a new directory in both data/ and dataraw/ for the update
		Usage: ./distributeFiles.sh <zipfile> [OPTION] (use option -h for help)
			-b: bypass debug mode (don't keep temp files)
			-s: output settings only
			-h: print help
